src/payment/payment-methods/cybersource/service.js

```diff
@@ -69,7 +81,16 @@ export async function checkout(basket, { cardHolderInfo, cardDetails }) {
         'Content-Type': 'application/x-www-form-urlencoded',
       },
     },
-  );
+  ).catch((error) => {
+    logAPIErrorResponse(error, {
+      messagePrefix: 'Cybersource Submit Error',
+      paymentMethod: 'Cybersource',
```

Comment:
Drying this in the future?  Maybe a helper like `getCybersourceErrorCustomAttributes(basketId, paymentErrorType)`.   It could always include `paymentMethod: 'Cybersource'`.  Or, if it is more general and takes `paymentMethod` as a param, this should take a constant for consistency.